lineman/spec/helpers/factory.js

```diff
@@ -0,0 +1,70 @@
+window.useFactory = function() {
```

Comment:
I really don't want it on window, but my javascript-fu isn't strong enough to patch it into Jasmine at the moment.